ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -42,6 +42,18 @@ spec:
       policyName: "subscriptions-policy"
     - fileName: StorageOperatorStatus.yaml
       policyName: "subscriptions-policy"
+    - fileName: AmqSubscriptionNS.yaml
+      policyName: "subscriptions-policy"
+    - fileName: AmqSubscriptionOperGroup.yaml
+      policyName: "subscriptions-policy"
+    - fileName: AmqSubscription.yaml
+      policyName: "subscriptions-policy"
+    - fileName: BareMetalEventRelaySubscriptionNS.yaml
+      policyName: "subscriptions-policy"
+    - fileName: BareMetalEventRelaySubscriptionOperGroup.yaml
+      policyName: "subscriptions-policy"
+    - fileName: BareMetalEventRelaySubscription.yaml
+      policyName: "subscriptions-policy"
```

Comment:
These should all be commented out (but don't delete the lines). Both Fast Event notification and BMER are optional content that users should opt in to.

Reply:
done